src/triggers.js

```diff
@@ -327,6 +327,7 @@ export function getResponseObject(request, resolve, reject) {
       response = {};
       if (request.triggerName === Types.beforeSave) {
         response['object'] = request.object._getSaveJSON();
+        response['object']['objectId'] = request.object.id;
```

Comment:
I would suggest to check `config.allowCustomObjectId` and to only set objectId if true

Reply:
No need - `RestWrite` constructor takes care of that

Reply:
@davimacedo How can I set custom object ids from javascript, given #1097 ?

Reply:
@davimacedo Test added